WordPress/src/main/java/org/wordpress/android/ui/CommentFullScreenDialogFragment.kt

```diff
@@ -55,11 +59,30 @@ class CommentFullScreenDialogFragment : Fragment(), CollapseFullScreenDialogCont
             reply.setText(it.getString(EXTRA_REPLY))
             reply.setSelection(it.getInt(EXTRA_SELECTION_START), it.getInt(EXTRA_SELECTION_END))
             viewModel.init()
+
+            setupSuggestionServiceAndAdapter(it.getSerializable(EXTRA_SITE_MODEL) as SiteModel, reply)
         }
 
         return layout
     }
 
+    private fun setupSuggestionServiceAndAdapter(mSite: SiteModel, reply: SuggestionAutoCompleteText) {
+        if (!isAdded || mSite == null || !SiteUtils.isAccessedViaWPComRest(mSite)) {
```

Comment:
Android Studio shows a warning/info for `mSite == null`  since `mSite` will never be `null`. It probably will after the change to `SITE_ID` though (please see my comment below).